packages/styles/src/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "moduleResolution": "Bundler",
     "noEmit": true,
     "skipLibCheck": true,
-    "target": "ESNext"
+    "target": "ESNext",
```

Comment:
May not need this.

Reply:
This may be required though. By telling TS we're aiming for the latest supported by TS version, we ensure it doesn't assume any polyfills to be added.

Reply:
I mean the extra comma. 🤣

Reply:
Fixed